.github/workflows/node.js.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
```

Comment:
Kimberly, I really like this action. Can you recreate it on the dev branch. We don't currently have any Node actions running and we need it. Thanks!